email-alert-api/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM ruby:2.6.3
+
```

Comment:
For consistency, it would be good to have `vim` installed here and to set the EDITOR env var - the usage is to enable the `bundle open` command that would normally be available.